src/app/components/OptimizelyPageMetrics/index.jsx

```diff
@@ -1,21 +1,42 @@
 import React, { useState, useContext, useEffect, useRef } from 'react';
 import { OptimizelyContext } from '@optimizely/react-sdk';
 import { RequestContext } from '#contexts/RequestContext';
+import useOptimizelyScrollDepth from '#hooks/useOptimizelyScrollDepth';
 import useOptimizelyVariation from '#hooks/useOptimizelyVariation';
 import OPTIMIZELY_CONFIG from '#lib/config/optimizely';
 
-const OptimizelyArticleCompleteTracking = () => {
+const OptimizelyPageMetrics = ({ pageView, pageDepth, pageComplete }) => {
   const ref = useRef();
   const observer = useRef();
   const { isAmp } = useContext(RequestContext);
   const { optimizely } = useContext(OptimizelyContext);
+  const [pageViewSent, setPageViewSent] = useState(false);
   const [pageCompleteSent, setPageCompleteSent] = useState(false);
   const [isVisible, setIsVisible] = useState(false);
 
   const experimentVariation = useOptimizelyVariation(OPTIMIZELY_CONFIG.flagKey);
+  const hasVariationKey = experimentVariation !== null;
 
   const sendPageCompleteEvent =
-    experimentVariation && !isAmp && !pageCompleteSent && isVisible;
+    pageComplete &&
+    experimentVariation &&
+    !isAmp &&
+    !pageCompleteSent &&
+    isVisible;
+
+  const sendPageViewEvent =
+    pageView && hasVariationKey && !isAmp && !pageViewSent;
+
+  useOptimizelyScrollDepth(pageDepth);
```

Comment:
I don't think we need to pass pageDepth as a param to the scroll depth function

```suggestion
  if (pageDepth) useOptimizelyScrollDepth();
```